src/test/java/com/hedera/exchange/ERTprocTestCases.java

```diff
@@ -302,12 +308,17 @@ public void testWeightedMedian() throws Exception {
         when(mockBinance.getHBarValue()).thenReturn(0.035);
         when(mockBinance.getVolume()).thenReturn(1000.0);
 
+        PayBito mockPayBito = mock(PayBito.class);
```

Comment:
where do you assert these values?

Reply:
This value is used to calculate the median. So we assert the median instead to see if the value is consistent with this change.